src/components/LoginModal.vue

```diff
@@ -266,14 +323,25 @@ export default {
             <q-separator/>
             <q-tab-panels v-model="tab" animated>
                 <q-tab-panel name="web3">
-                    <q-card class="c-login-modal__image-container" @click="injectedWeb3Login()">
+                    <q-card class="c-login-modal__image-container" @click="connectMetaMask()">
                         <q-img
                             :src="metamaskLogo"
-                            class="wallet-img"
                             height="64px"
                             width="64px"
                         />
-                        <p>{{ !browserSupportsMetaMask ? $t('components.continue_on_metamask') : 'Metamask' }}</p>
+                        <p>{{ isMobile ? $t('components.continue_on_metamask') : 'Metamask' }}</p>
+                    </q-card>
+                    <q-card
+                        v-if="isBraveBrowser"
+                        class="c-login-modal__image-container"
+                        @click="connectBraveWallet()"
+                    >
+                        <q-img
+                            :src="braveBrowserLogo"
+                            height="64px"
+                            width="64px"
```

Comment:
the logo is cut off for me, i think its because the brave logo (unlike MM) isnt actually square aspect ratio. try setting height to 64px and leave out the width

Reply:
good catch, resolved by omitting height and adjusting width to be closer in size to MM logo.